DigitalExcellence/dex-frontend
# Bugfix/97 new design illustrations

## Description

A few sentences describing the overall goals of the pull request's commits.
changing the images to copyright free images

## Type of change


## Checklist


## Steps to Test or Reproduce

Outline the steps to test or reproduce the PR here.
These steps will be used during release testing.

1. go to login and check the new image
2. go to the homepage and check the new image

## Link to issue
https://github.com/DigitalExcellence/dex-frontend/issues/97

Closes: #97
